spark/agent/src/main/scala/za/co/absa/spline/harvester/ModelConstants.scala

```diff
@@ -39,30 +39,30 @@ object ModelConstants {
     val Name = "name"
 
     // op.Read
-    val SourceType = "sourceType"
+    val SourceType = "sourceType" //todo: find better attribute name. See: https://github.com/AbsaOSS/spline/issues/496
 
     // op.Write
-    val DestinationType = "destinationType"
+    val DestinationType = "destinationType" //todo: find better attribute name. See: https://github.com/AbsaOSS/spline/issues/496
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Take the required action to fix the issue indicated by this comment.](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4600675)